.github/workflows/main.yml

```diff
@@ -0,0 +1,100 @@
+name: Run App Tests
```

Comment:
This is very minor (and largely personal preference), but with experience I've noticed that I tend to name GHA workflows more as "things" than "actions", i.e. here `App tests` (leaving the verb "Run" assumed).